Tests/run_tests.py

```diff
@@ -101,6 +101,7 @@ def is_numpy():
     "Bio.KEGG.Compound",
     "Bio.KEGG.Enzyme",
     "Bio.NMR.xpktools",
+    "Bio.MaxEntropy"
```

Comment:
Missing trailing comma - would instead get concatenated with the next line I think.